Snakefile

```diff
@@ -0,0 +1,496 @@
+from datetime import datetime
+from snakemake.remote.GS import RemoteProvider as GSRemoteProvider
+GS = GSRemoteProvider()
+
+# --- Settings --- #
+# Current date in YYYY-MM-DD format:
+timeStamp = datetime.now().strftime("%Y-%m-%d")
+
+# Configuration is read from the config yaml:
+configfile: 'configuration.yaml'
+logFile = f"{config['global']['logDir']}/evidence_parser.{timeStamp}.log"
+
+# --- All rules --- #
+rule all:
+    input:
+        GS.remote(f"{config['ClinGen']['outputBucket']}/ClinGen-{timeStamp}.json.gz"),
+        GS.remote(f"{config['PheWAS']['outputBucket']}/phewas_catalog-{timeStamp}.json.gz"),
+        GS.remote(f"{config['SLAPEnrich']['outputBucket']}/slapenrich-{timeStamp}.json.gz"),
+        GS.remote(f"{config['Gene2Phenotype']['outputBucket']}/gene2phenotype-{timeStamp}.json.gz"),
+        GS.remote(f"{config['CRISPR']['outputBucket']}/crispr-{timeStamp}.json.gz"),
+        GS.remote(f"{config['SysBio']['outputBucket']}/sysbio-{timeStamp}.json.gz"),
+        GS.remote(f"{config['Phenodigm']['outputBucket']}/phenodigm-{timeStamp}.json.gz")
+
+# --- Auxiliary Rules --- #
+## help               : prints help comments for Snakefile
+rule help:
+    input: "Snakefile"
+    shell:
+        "sed -n 's/^##//p' {input}"
+
+## clean               : prints help comments for Snakefile
+rule clean:
+    shell:
+        "rm -rf tmp"
+
+# --- Data sources parsers --- #
+## clingen          : processes the Gene Validity Curations table from ClinGen
+rule clingen:
+    input:
+        f"tmp/ClinGen-Gene-Disease-Summary-{timeStamp}.csv"
+    output:
+        evidenceFile=GS.remote(f"{config['ClinGen']['outputBucket']}/ClinGen-{timeStamp}.json.gz"),
+        unmappedDiseases=f"tmp/unmappedDiseases/clingen_unmapped_diseases-{timeStamp}.lst"
+    conda:
```

Comment:
Why do we need to specify the environment all the time? I thought we have an environment described by the yaml, and you run the entire pipeline on that env. I think it would be important if certain rules would be different requirements. But I might be wrong.

Reply:
I expected this to be necessary to address the particularities of the old Phenodigm environment. that's not the case anymore, so I've removed it.